public/app/features/alerting/unified/components/contact-points/utils.ts

```diff
@@ -50,6 +50,9 @@ export function getReceiverDescription(receiver: ReceiverConfigWithMetadata): Re
     case 'webhook': {
       return settings.url;
     }
+    case 'jira': {
+      return `Creates a "${settings.issue_type}" issue in the "${settings.project}" project`;
```

Comment:
Are `issue_type` and `project` required on the backend side? Just wondering if you can create one without these and end up with a "broken" looking description of the receiver 🤔

Reply:
I added a [commit](https://github.com/grafana/grafana/pull/100482/commits/62fef73b4013e7dca162a2df79fdcdfb336933ba) making them required